### ISSUE TITLE
SqueezeNet in PyTorch

### ISSUE THREAD

**Issue by Maratyszcza** [DeepScale/SqueezeNet#34]
SqueezeNet 1.0 and 1.1 are now available as a built-in model in the official [pytorch/vision](https://github.com/pytorch/vision) repo. In addition to model implementation, I pre-trained SqueezeNet 1.0 and 1.1 on ImageNet for PyTorch model zoo, and accuracy is even slightly better than the original Caffe models:

| Model          | Top-1 accuracy | Top-5 accuracy |
| -------------- |:--------------:|:--------------:|
| SqueezeNet 1.0 | 58.000%        | 80.488%        |
| SqueezeNet 1.1 | 58.184%        | 80.514%        |

Links: code [in the repo](https://github.com/pytorch/vision/blob/master/torchvision/models/squeezenet.py), discussion is [in the PR](https://github.com/pytorch/vision/pull/49)

**Comment by forresti** [comment]
Bravo!!!

**Comment by Maratyszcza** [comment]
@forresti Would you add a link to readme?

**Comment by forresti** [comment]
Done!

**Comment by Maratyszcza** [comment]
Thanks!
